bin/README.md

```diff
@@ -27,22 +27,22 @@ authentication method is under development).
 
 Support of TLS on the Server is managed through a pkcs12 der file. The documentation is captured in the example test config file, [example.toml](https://github.com/hickory-dns/hickory-dns/blob/main/tests/test-data/test_configs/example.toml). A registered certificate to the server can be pinned to the Client with the `add_ca()` method. Alternatively, as the client uses the rust-native-tls library, it should work with certificate signed by any standard CA.
 
-DoT and DoH are supported. This is accomplished through the use of one of `native-tls`, `openssl`, or `rustls` (only `rustls` is currently supported for DoH). The Resolver requires valid DoT or DoH resolvers being registered in order to be used.
+DoT and DoH are supported. This is accomplished through the use of `rustls`. The Resolver requires valid DoT or DoH resolvers being registered in order to be used.
 
 Client authentication/mTLS is currently not supported, there are some issues
 still being worked on. TLS is useful for Server authentication and connection
 privacy.
 
-To enable DoT, one of the features `dns-over-native-tls`, `dns-over-openssl`, or
-`dns-over-rustls` must be enabled. `dns-over-https-rustls` is used for DoH.
+To enable DoT, one of the features `dns-over-rustls` must be enabled. `dns-over-https-rustls` is used for DoH. `dns-over-h3` is used for dns over http3 and `dns-over-quic`
```

Comment:
Can you turn this into a bullet list, too?

Reply:
Updated